src/skinview2d.ts

```diff
@@ -0,0 +1,56 @@
+// import { loadSkinToCanvas } from "skinview-utils";
+
+class SkinvViewer {
```

Comment:
Maybe it's better to use `Skin2DViewer` to avoid conflicts?

Reply:
what about `SkinViewer2D`?